src/main/java/org/cryptomator/frontend/fuse/mount/EnvironmentVariables.java

```diff
@@ -1,18 +1,20 @@
 package org.cryptomator.frontend.fuse.mount;
 
+import org.cryptomator.frontend.fuse.FileNameTranscoder;
+
 import java.nio.file.Path;
 import java.util.Optional;
 
 public class EnvironmentVariables {
 
 	private final Path mountPoint;
+	private final Optional<FileNameTranscoder> fileNameTranscoder;
```

Comment:
Since it is easy to obtain a pre-configured fileNameTranscoder (that would otherwise be used anyway in case of empty optionals), this doesn't need to be optional.

Reply:
fixed in 217e0a0